pr_agent/settings/configuration.toml

```diff
@@ -10,6 +10,7 @@ use_repo_settings_file=true
 ai_timeout=180
 max_description_tokens = 500
 max_commits_tokens = 500
+max_model_tokens = 32000 # even if a model supports more tokens, quality may degrade. Hence, enable to limit the number of tokens.
```

Comment:
**Suggestion:** The 'max_model_tokens' setting should be documented to explain its purpose and effect on the system, especially since it overrides the model's default token limit.
```suggestion
# Limits the maximum number of tokens that can be used by any model, regardless of the model's default capabilities.
# This can be useful to ensure consistent performance and to prevent issues related to large token counts.
max_model_tokens = 32000
```